risedev.yml

```diff
@@ -24,7 +24,7 @@ profile:
     #   RW_ENABLE_PRETTY_LOG: "true"
     steps:
       # If you want to use the local s3 storage, enable the following line
-      # - use: minio
+      - use: minio
```

Comment:
Next time you can define your risedev profile in `risedev-profiles.user.yml` so that your risedev profile won't be included in the PR.

Reply:
My bad.